packages/requisitions/src/CustomerRequisition/DetailView/SidePanel.tsx

```diff
@@ -59,6 +61,24 @@ const RelatedDocumentsSection: FC<SidePanelProps> = () => {
   );
 };
 
+const DatesSection: FC<SidePanelProps> = ({ draft }) => {
```

Comment:
should this be here, now that the date is in the top bit?
If so - the heading should change 😄

Reply:
whoops 😁